go.mod

```diff
@@ -2,7 +2,7 @@ module go.flow.arcalot.io/engine
 
 go 1.21
 
-toolchain go1.21.6
+toolchain go1.22.2
```

Comment:
I think Matt and I went round and round trying to figure out whether this was good, bad, or indifferent...and, I can't remember where we came out.  (I _think_ we decided it was "mostly-harmless".)

But, my instinct is that, if we're requiring only Go `1.21`, then we shouldn't be letting the `.mod` file be generated by or for `1.22` (and, I don't think we're ready to upgrade to `1.22` for the base requirement, yet).

So, unless someone can convince me that it's silly, I think we should leave the toolchain alone until we're also ready to update the `go` line.